addons/l10n_ec/__manifest__.py

```diff
@@ -1,35 +1,78 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Ecuadorian Accounting',
+    'version': '3.3',
+    'description': '''
+Functional
+----------
 
-# Copyright (C) 2010-2012 Cristian Salamea Gnuthink Software Labs Cia. Ltda
+This module adds accounting features for Ecuadorian localization, which
+represent the minimun requirements to operate a business in Ecuador in compliance
+with local regulation bodies such as the ecuadorian tax authority -SRI- and the 
+Superintendency of Companies -Super Intendencia de Compañías-
 
-{
-    'name': 'Ecuador - Accounting',
-    'version': '1.1',
-    'category': 'Accounting/Localizations/Account Charts',
-    'description': """
-This is the base module to manage the accounting chart for Ecuador in Odoo.
-==============================================================================
+Follow the next configuration steps:
+1. Go to your company and configure your country as Ecuador
+2. Install the invoicing or accounting module, everything will be handled automatically
+
+Highlights:
+* Ecuadorian chart of accounts will be automatically installed, based on example provided by Super Intendencia de Compañías
+* List of taxes (including withholds) will also be installed, you can switch off the ones your company doesn't use
+* Fiscal position, document types, list of local banks, list of local states, etc, will also be installed
 
-Accounting chart and localization for Ecuador.
-    """,
-    'author': 'Gnuthink Co.Ltd.',
+Technical
+---------
+Master Data:
+* Chart of Accounts, based on recomendation by Super Cías
+* Ecuadorian Taxes, Tax Tags, and Tax Groups
+* Ecuadorian Fiscal Positions
+* Document types (there are about 41 purchase documents types in Ecuador)
+* Identification types
+* Ecuador states with its codes
+* Ecuador banks
+* Partners: Consumidor Final, SRI, IESS, and also basic VAT validation
+
+Authors:
+    Ing. Andres Calle <andres.calle@trescloud.com>
+    Ing. José Miguel Rivero <jose.rivero@trescloud.com>
+    ''',
+    'author': 'OPA CONSULTING & TRESCLOUD',
+    'category': 'Localization',
```

Comment:
because 14 is stable between a moment. If we add fields, we will need an extra module for it. Which is not necessarily a problem. 
-> Module added , called l10n_ec_edi (name is open to changes)

fiscal positions are defined in different ways, but I have not encountered any mappings for the taxes/accounts. In other localizations we define a field on the partner to indicate the responsibility type. 
-> As explained in meetings, Fiscal position mapping with taxes in dependent on many factors(company type, industry type, among others), and is a per company configuration done at implementation.

tax groups are defined twice
-> Tax groups are defined in l10n_ec, and completed in l10n_ec_edi

there is a tax report model in community that we can define here instead of defining tags of type taxes, but I have no idea what this report looks like for Ecuador
-> The tax "report" is a web form on the SRI website, as per the following screen shot:
![image](https://user-images.githubusercontent.com/112407/110168735-96fb1a00-7dc5-11eb-92d2-a66b19c11384.png)

no need to add an internal library
-> removed

A lot of taxes are defined and we need to see if these are really taxes for daily use. An accountant could create the tax himself as well instead of trying to define everything upfront.
-> This is already a filtered list, less common items are inactive, and can be enabled if needed.